packit_service/service/api/testing_farm.py

```diff
@@ -134,3 +137,40 @@ def validate_testing_farm_request():
         msg = "Invalid testing farm secret provided"
         logger.warning(msg)
         raise ValidationFailed(msg)
+
+
+@ns.route("/get-results")
+class TestingFarmResultsList(Resource):
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT, "Testing Farm Results list follows")
+    def get(self):
+        """ List all Testing Farm  results. """
+
+        result = []
+
+        first, last = indices()
+        # results have nothing other than ref in common, so it doesnt make sense to
+        # merge them like copr builds
+        for tf_result in TFTTestRunModel.get_all(first, last):
+
+            result_dict = {
+                "pipeline_id": tf_result.pipeline_id,
+                "ref": tf_result.commit_sha,
+                "status": tf_result.status,
+                "target": tf_result.target,
+                "web_url": tf_result.web_url,
+                "pr_id": tf_result.get_pr_id(),
+            }
+
+            project = tf_result.get_project()
+            if project:
```

Comment:
When will this happen?

Reply:
I think almost always. If project is a check to see if its not None / exists. It might be redundant because a Test Result will always have a git project associated with it

Reply:
Okay, sorry, wrong question :) I wanted to ask when 'project' will be not true-ish :)

So if this is expected to be redundant, then let's remove it.